teams/wg-rls-2.toml

```diff
@@ -3,7 +3,7 @@ subteam-of = "compiler"
 kind = "working-group"
 
 [people]
-leads = ["matklad"]
```

Comment:
I'd maybe removed @matklad while are we at it -- he has been slacking these past few months!